src/ATen/native/xpu/sycl/AbsKernel.h

```diff
@@ -4,6 +4,6 @@
 
 namespace at::native::xpu {
 
-void abs_kernel(TensorIteratorBase& iter);
+TORCH_XPU_API void abs_kernel(TensorIteratorBase& iter);
```

Comment:
We expose the symbol because of the current solution. In the future, it should be not necessary if we combine the libraries as a single one as long as device image compression is ready. So I suggest replacing `TORCH_XPU_API` with another name like `TORCH_XPU_API_MSVC`. In the future, we can easily remove `TORCH_XPU_API_MSVC` by searching and replacing.

Reply:
I see your concerns. I suggest keeping it,
1. As for replacement in future, the changes all are included in `XXXKernel.h`. It is easy to find out them.
2. As for correct logic of these macros, basically TORCH_XPU_API is designed and includes the compatibility on Windows on Linux. Add a new one let people confused, like some people would say why could `TORCH_XPU_API` not do this, or is there new concept involved in the new macro.